### PR TITLE
docs: add docs/operating/manuals/resourceconsist.md

### PR DESCRIPTION
**Pull request by WeichengWang1**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [x] N
- [ ] Y 

#### 2. What is the scope of this PR (e.g. component or file name):
docs/operating/manuals/

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [x] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [x] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: docs/operating/manuals/resourceconsist.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+**Employer** is the entity responsible for managing and coordinating the utilization of another resource, similar to how a service selects and controls pods.
+
+Employer can be any kind, and CRD is of course can be used as Employer.
+### Employee
+**Employee** is the resource managed by another resource, like pods selected by service.
+
+Same with Employer, Employee can be any kind, and CRD is of course can be used as Employee.
+
+If an adapter implementing ReconcileAdapter and following PodOpsLifecycle, the Employee should be Pod.
+## Key Interface/Struct Definition
```
Comment (wu8685) [inline docs/operating/manuals/resourceconsist.md:22]: Definition`s`?

File: docs/operating/manuals/resourceconsist.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+### ReconcileAdapter
+**ReconcileAdapter** is an interface specifying a set of methods as follows.
```
Comment (wu8685) [inline docs/operating/manuals/resourceconsist.md:24]: moving codes under this line would be better

File: docs/operating/manuals/resourceconsist.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+	b := md5.Sum([]byte(employerName))
+	return v1alpha1.PodOperationProtectionFinalizerPrefix + "/" + hex.EncodeToString(b[:])[8:24]
+}
+```
+## Key Finalizers
+### LifecycleFinalizer
+**LifecycleFinalizer** is a finalizer on Employee used to following PodOpsLifecycle, removed in preparing period of PodOpsLifecycle and added in completing period of PodOpsLifecycle
+```
+const (
+	PodOperationProtectionFinalizerPrefix = "prot.podopslifecycle.kusionstack.io"
```
Comment (wu8685) [inline docs/operating/manuals/resourceconsist.md:133]: It should be highlighted that the prefix is required.

**Comment by WeichengWang1** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**